resources/psi-ms-PARSED.xml

```diff
@@ -193,6 +195,7 @@
     <instrument id="MS:1000641" name="DSQ II" vendor="Thermo Scientific" description="Thermo Scientific DSQ II GC-MS."/>
     <instrument id="MS:1000449" name="LTQ Orbitrap" vendor="Thermo Scientific" description="Finnigan LTQ Orbitrap MS."/>
     <instrument id="MS:1000640" name="DFS" vendor="Thermo Scientific" description="Thermo Scientific DFS HR GC-MS."/>
+    <instrument id="MS:1003029" name="Orbitrap Eclipse" vendor="Thermo Scientific" description="&quot;Thermo Scientific Orbitrap Eclipse mass spectrometer with Tribrid architecture consisting of quadrupole mass filter, linear ion trap and Orbitrap mass analyzers.&quot; [PSI:PI]"/>
```

Comment:
I take that back! Looks like we've got an extra quote encoded in the description here.